clustering/consul.py

```diff
@@ -81,6 +81,11 @@
           - the port on which the service is listening required for
             registration of a service, i.e. if service_name or service_id is set
         required: false
+    service_address:
+        description:
+          - the address on which the service is serving required for
+            registration of a service
+        required: false
```

Comment:
please add the following to the new param docs:
~~~
default: localhost
version_added: "2.1"
~~~